core/deployment/src/main/java/io/quarkus/deployment/dev/testing/JunitTestRunner.java

```diff
@@ -42,11 +42,7 @@
 import org.junit.jupiter.api.TestTemplate;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.platform.commons.annotation.Testable;
-import org.junit.platform.engine.FilterResult;
-import org.junit.platform.engine.TestDescriptor;
-import org.junit.platform.engine.TestExecutionResult;
-import org.junit.platform.engine.TestSource;
-import org.junit.platform.engine.UniqueId;
+import org.junit.platform.engine.*;
```

Comment:
The only issue is that we got back to star imports here. I tried to push to your branch but can't.

Could you adjust (and ideally configure your IDE to avoid star imports)?

Reply:
Here is the diff if you need help: 

```suggestion
import org.junit.platform.engine.FilterResult;
import org.junit.platform.engine.TestDescriptor;
import org.junit.platform.engine.TestExecutionResult;
import org.junit.platform.engine.TestSource;
import org.junit.platform.engine.UniqueId;
```

Reply:
Yes, you're right. I disabled star imports right after the first commits. Thanks for the hints.